_posts/2020-02-22-gsoc2020.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: Google Summer of Code 2020
+---
+
+<p>
+<img
+src="https://developers.google.com/open-source/gsoc/images/gsoc2016-sun-373x373.png"
+title="Google Summer of Code 2020" alt="Google Summer of Code 2020"
+style="float: left; height: 4.5em; " />
+</p>
+
+MDAnalysis has been accepted as an [organization][org] into [Google Summer of Code 2020][gsoc]. If you are interested in working with us
+this summer as a student read the advice and links below and write to us on the
```

Comment:
This sentence needs a comma, can I suggest: "If you are interested in working with us as a student this summer, ...."?